index.js

```diff
@@ -163,6 +163,8 @@ const KEYWORDS = {
 const MAX_WORDS_ANALYZED = 9999;
 // Max number of keywords to send
 const MAX_KEYWORDS = 10;
+// Minimum number of occurrences of a keyword to consider it
+const MIN_KEYWORD_OCCURRENCES = 2;
```

Comment:
I'd love to see a `debug` log message about this (keywords & count) in the console, if that isn't too intrusive for users. Could be useful.

Reply:
This is easy enough to do. I'm logging it at debug/verbose so you may have to turn your logging up.

![Screenshot from 2021-04-15 15-46-52](https://user-images.githubusercontent.com/185043/114947399-f0f9f100-9e01-11eb-9d39-ece220678bd5.png)